readme.md

```diff
@@ -21,8 +21,8 @@ Follow
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://VladyslavIlienko.github.io/layout_stars/)
+- [TEST REPORT LINK](https://VladyslavIlienko.github.io/layout_stars/report/html_report/)
```

Comment:
provide these links in the PR description and fix them

feel free to ask for some help in the chat

Reply:
- [DEMO LINK](https://VladyslavIlienko.github.io/layout_stars/)
- [TEST REPORT LINK](https://VladyslavIlienko.github.io/layout_stars/report/html_report/)